style.css

```diff
@@ -0,0 +1,1210 @@
+  display: none;
+}
+
+.about-partner-container {
+  max-width: 100rem;
+  margin: 0 auto;
+  padding: 1.6rem;
+  width: 100vw;
+}
+
+.about-partner-container h2 {
+  margin: 0 auto;
+  text-align: center;
+  font-size: 2.5rem;
+  font-weight: 300;
+  color: var(--mobile-secondary-color);
+  padding-top: 2rem;
+}
+
+.about-partner-logos {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 4rem;
+  margin-top: -5rem;
+}
+
+.about-footer-section {
+  background: var(--primary-color);
+}
+
+.about-footer-container {
+  max-width: 100rem;
+  margin: 0 auto;
+  padding: 4rem;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 5rem;
+}
+
+.about-footer-logo {
+  width: 80px;
+  height: 50px;
+  border-radius: 2px;
+}
+
+.footer-logo-mobile {
+  display: none;
+}
+
+.about-footer-text {
+  font-size: 1.4rem;
+  color: var(--secondary-color);
+}
+
+@media screen and (max-width: 768px) {
+  /* ===============HEADER==================== */
+  .page-info {
+    display: none;
+  }
+
+  .nav-bar {
+    display: flex;
+    background: transparent;
+    position: absolute;
+    top: 0;
+    margin-top: 0;
+    box-shadow: none;
+    justify-content: flex-start;
+  }
+
+  .nav-logo {
+    display: none;
+  }
+
+  .nav-links {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    background: rgba(236, 83, 66, 0.93);
+    -webkit-backdrop-filter: blur(5px);
+    backdrop-filter: blur(5px);
+    height: 100vh;
+    width: 250px;
+    top: 0;
+    left: -250px;
+    text-align: left;
+    z-index: 2;
+    transition: 1s;
+  }
+
+  .nav-items {
+    flex-direction: column;
+    padding-top: 7rem;
+    padding-left: 10rem;
+    line-height: 5rem;
+    letter-spacing: 3px;
+  }
+
+  .nav-items li a {
+    color: var(--mobile-secondary-color);
+    font-size: 2rem;
+  }
+
+  .btn {
+    display: flex;
+    justify-content: center;
+    border: 3px solid var(--mobile-secondary-color);
+    color: var(--mobile-secondary-color);
+    max-width: 50%;
+    padding: 1rem 3rem;
+    margin-top: 2rem;
+    white-space: nowrap;
+  }
+
+  .btn-margin {
+    margin-left: 6rem;
+  }
+
+  .btn:hover {
+    background-color: var(--accent-color);
+  }
+
+  .fas.fa-times {
+    display: none;
+    color: var(--primary-color);
+    margin: 1.5rem;
+    position: absolute;
+    margin-left: 125vw;
+    padding-left: 0.75rem;
+    font-size: 2.2rem;
+    cursor: pointer;
+  }
+
+  .fas.fa-bars {
+    display: block;
+    color: var(--primary-color);
+    font-size: 2.8rem;
+    position: absolute;
+    left: 1rem;
+    cursor: pointer;
+  }
+
+  /* ========LANDING PAGE========== */
+  .landing-page {
+    padding-top: 7rem;
+  }
+
+  .text-box h1 {
+    font-size: 1.5rem;
+  }
+
+  .text-box p {
+    font-size: 5.5rem;
+    margin-bottom: 0;
+  }
+
+  .year-text {
+    font-size: 4.5rem;
+  }
+
+  .subtext {
+    width: 61% !important;
+    margin-top: 10px;
+    text-align: center;
+  }
```

Comment:
- [ ] [OPTIONAL] I would suggest to keep the width to 100% on mobile devices, given the original design it would me more appropriate, what you have currently :point_down: 
![image](https://user-images.githubusercontent.com/97521557/219447791-14262536-d213-4a63-8713-6c2f65c29cf5.png)

original design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/219448070-688550b8-7f40-4681-ab1d-ff3c157b866d.png)

Reply:
thank you implemented